src/demo/demoUtils.ts

```diff
@@ -6,6 +6,8 @@ import {
   StepTypes,
 } from '~types/steps'
 import { ServerRegions, SdkOptions } from '~types/sdk'
+// @ts-ignore
+import testCobrandLogo from './assets/onfido-logo.svg'
```

Comment:
Minor: IMO the test cobrand logo image should be something other than the logo that's also used for the existing text only cobranding feature. But appreciate that this also means sourcing a suitable design asset.

Reply:
Yeah, I can definitely see that, but I feel making it point to a separate file, even if it's the same image, should be enough coming from a testing perspective. My main concern with using a random image from somewhere is I don't want to create any kind of liability concerns from using an image without permission. If you can think of a good image we can use in it's place we have approval to use, please let me know and I'll switch them out.